contentcuration/contentcuration/static/js/edit_channel/clipboard/hbtemplates/clipboard_step_2_channel.handlebars

```diff
@@ -0,0 +1,70 @@
+<div id="clipboard" class="pull-right">
+	<div class="clipboard_header">
+		<h3 class="text-center">
+			<img src="/static/img/add_folder.png" alt="add" class="pull-left"/>
+			Adding Content to Topic
+			<a class="back pull-right btn-link toggle_clipboard">X</a>
+		</h3>
+		<table>
+			<tr>
+				<td class="text-center">1<h6>Source</h6></td>
+				<td class="text-center"><canvas></canvas></td>
+				<td class="text-center"><h4 class="text-center">2</h4><h6>Choose Content</h6></td>
+				<td class="text-center"><canvas></canvas></td>
+				<td class="text-center">3<h6>Metadata</h6></td>
+			</tr>
+		</table>
+	</div>
+	<div class="clipboard_content">
+		<label>2. Adding content from a channel</label>
```

Comment:
Having numbers at both levels of the list hierarchy is potentially confusing.

Again, ordered lists (and nesting them), might help with this.